README.md

```diff
@@ -12,4 +12,51 @@ Binary package
 
 ## Usage
 
-*TODO: Add instructions for usage*
+```bash
+$ hab pkg install core/gawk --binlink
```

Comment:
I'd suggest it's worthwhile leaving the hab printout here - otherwise all you see is the `--help` output.

Reply:
Amended differently but after pattern of flex example.